doc/lsst.analysis.tools/getting-started.rst

```diff
@@ -367,6 +365,10 @@ Adding an Action
 
 Actions go in one of the sub folders of the actions directory depending on what type they are, this is covered in the package layout section. Before you add a new action check if it is already included before adding a duplicate. Sometimes it will probably be better to generalise an exisiting action rather than making a new one that is very similar to something that already exists. If the new action is long or specific to a given circumatance then add it to a new file, for example the ellipticity actions in `python/lsst/analysis/tools/actions/vector/ellipticity.py <https://github.com/lsst/analysis_tools/blob/main/python/lsst/analysis/tools/actions/vector/ellipticity.py>`__.
 
+The current actions that are available are detailed :doc:`here<action-types>`. Most common requests are already coded up and
+please try to reuse actions that already exist before making your own. Please also try to make actions as
+reusable as possible so that other people can also use them.
```

Comment:
General comment: if you structure your ReST with the philosophy of one sentence per line, it will make future edits easier to parse.